### PR TITLE
Add events section to hackathons

### PR DESCRIPTION
**Pull request by zoek1**
##### Description
- Retrieve events from the provider and include one endpoint to don't expose AddEvent tokens
- Add a tab to display events in the hackathon view
- Add AddEvent button to provide an easy way to add events to different providers
- Change event style based on the events type

https://www.loom.com/share/a82ef2081ffb45e6bef233fe9214112d

##### Refers/Fixes
https://github.com/gitcoinco/web/issues/6695

### CODE REVIEW

File: app/dashboard/views.py [hunk 1]
```diff
@@ -5836,3 +5837,36 @@ def showcase(request, hackathon):
     return JsonResponse({
         'success': True,
     })
+
+
+def events(request, hackathon):
+    hackathon_event = get_object_or_404(HackathonEvent, id=hackathon)
```
Comment (octavioamu) [inline app/dashboard/views.py:5843]: as this seems to be a Rest endpoint with json answers lets keep the server answers to rest json payloads. In that way we can be sure the reason why failing is actually a not found event.

**Review by octavioamu — CHANGES_REQUESTED** [review]

**Review by PixelantDesign — APPROVED** [review]
looking good! @connoroday @jdorfman please take a look!

**Review by jdorfman — APPROVED** [review]
great work =)

**Review by frankchen07 — APPROVED** [review]
LGTM. Added a section on product documentation.

1. Sign in AddEvent
2. Add the Calendar ID from AddEvent to the hackathon event in the administration.
3. Opening a new AddEvent Calendar event will populate the hackathon event tab with the new calendar link.

**Comment by frankchen07** [comment]
also, do we have GA impression tracking for the event page?

File: app/dashboard/views.py (hunk 1 above)
Comment (zoek1) [reply app/dashboard/views.py]: done

**Comment by thelostone-mc** [comment]
@zoek1 Could you get travis to go green ?

**Comment by zoek1** [comment]
The only problem is the ipfs version, but i didn't touch such code :thinking:

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**